onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -286,56 +290,38 @@ def time_fn(args, fn, inputs):
         outputs = fn(inputs)
         logger.info(outputs)
 
-    input_sync = (  # noqa: E731
-        lambda *kwargs: args.io_binding.synchronize_inputs()
-        if args.device != "cpu" and args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}  # ORT synchronize
-        else lambda *kwargs: torch.cuda.synchronize()
-        if args.device != "cpu" and torch.cuda.is_available()  # PyTorch synchronize
-        else lambda *kwargs: None  # no-op function
-    )
-
-    output_sync = (  # noqa: E731
-        lambda *kwargs: args.io_binding.synchronize_outputs()
-        if args.device != "cpu" and args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}  # ORT synchronize
-        else lambda *kwargs: torch.cuda.synchronize()
-        if args.device != "cpu" and torch.cuda.is_available()  # PyTorch synchronize
-        else lambda *kwargs: None  # no-op function
-    )
-
     for _ in warmup_range:
-        input_sync()
         fn(inputs)
-        output_sync()
 
     # Benchmark
-    total_time = 0
+    if args.device != "cpu":
+        torch.cuda.synchronize()
+    start_time = time.time()
+
     bench_range = (
         range(args.num_runs)
         if args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}
         else trange(args.num_runs, file=sys.stdout, desc="Benchmark")
     )
     for _ in bench_range:
-        input_sync()
-        start_time = time.time()
-
         fn(inputs)
 
-        output_sync()
-        end_time = time.time()
-
-        total_time += end_time - start_time
+    if args.device != "cpu":
+        torch.cuda.synchronize()
+    end_time = time.time()
 
     # Newline print after trange in order to print metrics on new lines without progress bar on same line
     if args.benchmark_type not in {"ort-msft", "ort-convert-to-onnx"}:
         logger.info("")
 
-    latency = total_time / args.num_runs
+    latency = (end_time - start_time) / args.num_runs
     throughput = args.batch_size / latency
 
-    logger.info(f"Batch Size: {args.batch_size}")
-    logger.info(f"Sequence Length: {args.sequence_length}")
-    logger.info(f"Latency: {latency} s")
-    logger.info(f"Throughput: {throughput} tps")
+    if args.rank == 0:
+        logger.info(f"Batch Size: {args.batch_size}")
+        logger.info(f"Sequence Length: {args.sequence_length}")
```

Comment:
Is `args.rank = 0` for the 7B and 13B models? If not, this will break the `benchmark_all.py` script. The `benchmark_all.py` script relies upon printing all of these metrics.

Reply:
Yes, for single gpu case rank=0 and world_size=1